src/MovieForm.js

```diff
@@ -0,0 +1,59 @@
+export default function MovieForm({
+  titleForm,
+  setTitleForm,
+  directorForm,
+  setDirectorForm,
+  yearForm,
+  setYearForm,
+  colorForm,
+  setColorForm,
+  addMovie
+}) {
+
+  function handleSubmit(e) {
```

Comment:
to keep your MovieForm component clean, this handleSubmit should be defined in App.js and be passed to MovieForm as a prop